[vtk] Add config header for VTK::metaio

This is usually generated by CMake, but it's easier to just write it
by hand for our Bazel build.

TODO(jwnimmer-tri) Enhance settings.bzl to be able to express this
directly.

--- /dev/null
+++ Utilities/MetaIO/metaIOConfig.h
@@ -0,0 +1,2 @@
+#pragma once
+#define METAIO_FOR_VTK
